ArduCopter/land_detector.cpp

```diff
@@ -70,9 +70,13 @@ void Copter::update_land_detector()
         const bool landing = flightmode->is_landing();
         bool motor_at_lower_limit = (flightmode->has_manual_throttle() && (motors->get_below_land_min_coll() || heli_flags.coll_stk_low) && fabsf(ahrs.get_roll()) < M_PI/2.0f)
                                     || ((!force_flying || landing) && motors->limit.throttle_lower && pos_control->get_vel_desired_cms().z < 0.0f);
+        bool motor_at_minimum_mix = true;
 #else
         // check that the average throttle output is near minimum (less than 12.5% hover throttle)
-        bool motor_at_lower_limit = motors->limit.throttle_lower && attitude_control->is_throttle_mix_min();
+        bool motor_at_lower_limit = motors->limit.throttle_lower;
+        // throttle is never at mix min in airmode
+        bool motor_at_minimum_mix = attitude_control->is_throttle_mix_min() ||
```

Comment:
Randy is a the naming expert. But the param is called `motor_at_minimum_mix` it returns true in airmode but the comment says it is never at min in airmode, so one would expect it to be false in that case.

Reply:
Thanks fixed